test/test_queries.py

```diff
@@ -21,9 +21,8 @@
 @pytest.fixture
 def config():
     config = Configuration()
-    config.debug = False
     config.fmt = 'json'
```

Comment:
So just to clarify, leave this part as is for this current commit but work on format 'table' instead of 'json' for the next commit? 

Thanks for the feedback! I shall get the above sorted first.
Cheers.

Reply:
Yeah, just leave it for now, but I think it would be more beneficial to test the `table` formatter than the `json` one (I guess it would be most beneficial to test both, but then we're going to start making the `.test` files bigger and bigger and I'm not sure it's worth it.